### DISCUSSION TITLE
Some feature ideas

### DISCUSSION THREAD

**Discussion by MickMorucci** [BTCWebApp/BTCWeb.App#5] [Ideas]
Some features that would be cool to have, based on our conversations on twitter
category tags for each app (news, finance, crowdfunding)

High priority:
- [ ] Make it into a PWA: make it feel like an app from the get go.
- [x] app description 

Medium priority:
- [ ] rating the app. 
- [ ] comment.
- [ ] nostr connect.
- [ ] Zaps (anti-spam)
- [ ] number of downloads or clicks: show activity and engagement around different PWAs.

Tweet thread reference: https://twitter.com/metamick14/status/1669254043632377856?s=20

**Comment by SuperPhatArrow** [comment]
I was thinking we could also use [lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) scores in the build process to add a quality variable to the rating system as a balance to prevent gaming the rating system.

**Reply by MickMorucci** [reply]
We can prevent rigging of the rating system by using the ratings that your friends have given to an app - based on nostr npubs

**Reply by SuperPhatArrow** [reply]
I haven't done a ton of work on nostr, but as I understand it, the relay protocol design makes it difficult to see who follows whom with 100% certainty. If it can be done, then a web of trust part of the equation sounds like another great counterbalance, great thinking!
